UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -781,7 +787,46 @@ <h4>Metadata techniques</h4>
 					</ul>
 				</section>
 			</section>
+<section id="legal-considerations">
+<h3>Legal considerations</h3>
+<div class="note">
+					<p>This key information can be hidden if metadata is missing.</p>
```

Comment:
"missing" here seems to imply that the publisher forgot to include this information.  Maybe "not present"?